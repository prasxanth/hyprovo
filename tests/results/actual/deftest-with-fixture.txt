2021-11-24 22:51:24.617 - INFO - Pass ✓... (deftest-with-fixture / test-add-variable): sum should be 2
2021-11-24 22:51:24.619 - ERROR - Fail ✗... (deftest-with-fixture / test-add-variable): sum should be 3
2021-11-24 22:51:24.623 - INFO - Pass ✓... (deftest-with-fixture / test-sub-variable): difference should be 0
2021-11-24 22:51:24.625 - ERROR - Fail ✗... (deftest-with-fixture / test-sub-variable): difference should be 4
2021-11-24 22:51:24.629 - INFO - Pass ✓... (deftest-with-fixture / test-mul): mul pass
2021-11-24 22:51:24.631 - ERROR - Fail ✗... (deftest-with-fixture / test-mul): mul fail
2021-11-24 22:51:24.634 - INFO - Pass ✓... (deftest-with-fixture / test-div): div pass
2021-11-24 22:51:24.636 - ERROR - Fail ✗... (deftest-with-fixture / test-div): div fail
2021-11-24 22:51:24.636 - DEBUG - (deftest-with-fixture): Test Summary → Total: 8    Pass: 4    Fail: 4
