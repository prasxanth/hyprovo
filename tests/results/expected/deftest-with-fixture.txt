2021-11-24 22:50:31.458 - INFO - Pass ✓... (deftest-with-fixture / test-add-variable): sum should be 2
2021-11-24 22:50:31.461 - ERROR - Fail ✗... (deftest-with-fixture / test-add-variable): sum should be 3
2021-11-24 22:50:31.465 - INFO - Pass ✓... (deftest-with-fixture / test-sub-variable): difference should be 0
2021-11-24 22:50:31.468 - ERROR - Fail ✗... (deftest-with-fixture / test-sub-variable): difference should be 4
2021-11-24 22:50:31.472 - INFO - Pass ✓... (deftest-with-fixture / test-mul): mul pass
2021-11-24 22:50:31.475 - ERROR - Fail ✗... (deftest-with-fixture / test-mul): mul fail
2021-11-24 22:50:31.480 - INFO - Pass ✓... (deftest-with-fixture / test-div): div pass
2021-11-24 22:50:31.483 - ERROR - Fail ✗... (deftest-with-fixture / test-div): div fail
2021-11-24 22:50:31.483 - DEBUG - (deftest-with-fixture): Test Summary → Total: 8    Pass: 4    Fail: 4
